meson_options.txt

```diff
@@ -0,0 +1,27 @@
+option('cli', type: 'boolean', value: true, description: 'Build the CLI')
+option('inline-all', type: 'boolean', value: false,
+    description: 'Make all functions inline')
+option('namespace', type: 'string', value: '',
+    description: 'Prefixes all symbols with the value of XXH_NAMESPACE')
+option('force-memory-access', type: 'combo', choices: ['default', 'attribute', 'unaligned-reads', 'byteshift'],
+    value: 'default', description: 'See XXH_FORCE_MEMORY_ACCESS')
+option('force-align-check', type: 'boolean', value: false,
+    description: 'Use a faster direct read path when input is aligned')
+option('vector', type: 'combo', choices: ['default', 'scalar', 'sse2', 'avx2', 'avx512', 'neon', 'vsx'],
+    value: 'default', description: 'Manually select a vector instruction set')
+option('prefetch', type: 'boolean', value: true,
+    description: 'Disable prefetching. XXH3 only')
+option('prefetch-dist', type: 'integer', value: 0,
+    description: 'Select prefecting distance. XXH3 only')
```

Comment:
typo

Reply:
oof copy and past from the official README. good catch. fixing now